src/TensorValues/TensorValues.jl

```diff
@@ -78,7 +78,7 @@ export indep_components_names
 
 import Base: show
 import Base: zero, one
-import Base: +, -, *, /, \, ==, ≈, isless
+import Base: +, -, *, /, \, ==, ≈, isless, isapprox
```

Comment:
Not needed, `isapprox === ≈` is `true`.

Reply:
If removed, I get an error shouldimport Base.isapprox